pctiler/pctiler/endpoints/pg_mosaic.py

```diff
@@ -39,13 +39,16 @@ def __init__(self, request: Request):
     reader_dependency=ReaderParams,
     router_prefix=get_settings().mosaic_endpoint_prefix,
     backend_dependency=BackendParams,
+    add_map_viewer=False,
+    add_statistics=False,
+    add_mosaic_list=False,
```

Comment:
those are already set to False in the TileFactory so it's mostly informative
https://github.com/stac-utils/titiler-pgstac/blob/master/titiler/pgstac/factory.py#L111-L116